.github/settings.yml

```diff
@@ -63,6 +63,16 @@ branches:
 
 # Labels: define labels for Issues and Pull Requests
 labels:
+  - name: auro-accordion
+    cat: default
+    color: fa23e4
+    description:
+    process: Apply to issues to identify project repo
+  - name: Release
+    cat: default
+    color: 1c872e
+    description: Defines release scope of work
+    process: Apply to any issue that is the designated parent issue for a release
```

Comment:
This should not be here. It appears that you merged this commit into the last commit on the repo.